### PR TITLE
First pass at fixing style/security

### PR DESCRIPTION
**Pull request by insacuri**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,8 +3,11 @@
 build:
 	pipenv install --dev
 
-test:
-	pipenv check --style ./ras_backstage ./test
+lint:
+	pipenv run flake8 ./ras_backstage ./test
+	pipenv check ./ras_backstage ./test
```
[the remark this answers is not in the document]
Comment (andrewmil) [reply Makefile]: When we have that step in it `make test` doesn't run for me but without it it seems to work fine

[the remark this answers is not in the document]
Comment (insacuri) [reply Makefile]: flake8 doesn't check the packages for security vulnerabilities (of which there are some)

[the remark this answers is not in the document]
Comment (andrewmil) [reply Makefile]: Ah so is that step failing because of the security vulnerabilities?

[the remark this answers is not in the document]
Comment (insacuri) [reply Makefile]: Yes, but travis is still doing a style check then running the tests without doing pipenv check which is why it's passing here.

**Pull request merged**